presto-hive/src/test/java/com/facebook/presto/hive/metastore/TestMetastoreUtil.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.hive.metastore;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.apache.hadoop.hive.metastore.MetaStoreUtils;
+import org.apache.hadoop.hive.metastore.api.FieldSchema;
+import org.apache.hadoop.hive.metastore.api.Order;
+import org.apache.hadoop.hive.metastore.api.SerDeInfo;
+import org.apache.hadoop.hive.metastore.api.StorageDescriptor;
+import org.testng.annotations.Test;
+
+import java.util.Properties;
+
+import static org.testng.Assert.assertEquals;
+
+public class TestMetastoreUtil
+{
+    private static final StorageDescriptor TEST_STORAGE_DESCRIPTOR = new StorageDescriptor(
+            ImmutableList.of(
+                    new FieldSchema("col1", "bigint", "comment1"),
+                    new FieldSchema("col2", "binary", null),
+                    new FieldSchema("col3", "string", null)),
+            "hdfs://VOL1:9000/db_name/table_name",
+            "com.facebook.hive.orc.OrcInputFormat",
+            "com.facebook.hive.orc.OrcOutputFormat",
+            false,
+            100,
+            new SerDeInfo("table_name", "com.facebook.hive.orc.OrcSerde", ImmutableMap.of("sdk1", "sdv1", "sdk2", "sdv2")),
+            ImmutableList.of("col2", "col3"),
+            null,
+            ImmutableMap.of());
+    private static final org.apache.hadoop.hive.metastore.api.Table TEST_TABLE = new org.apache.hadoop.hive.metastore.api.Table(
+            "table_name",
+            "db_name",
+            "owner_name",
+            0,
+            0,
```

Comment:
The "0"s in this tests are all intentional because these fields are ignored and they don't round-trip.